src/gcp/firestore.ts

```diff
@@ -1,18 +1,16 @@
-"use strict";
+import api = require("../api");
 
-var api = require("../api");
-
-var _API_ROOT = "/v1beta1/";
+const _API_ROOT = "/v1beta1/";
 
 /**
  * List all collection IDs.
  *
  * @param {string} project the Google Cloud project ID.
```

Comment:
In the future, feel free to remove types from the JSDocs as the types are documented in the function definitions!